src/plugins/data/public/actions/filters/create_filters_from_value_click.ts

```diff
@@ -140,5 +141,5 @@ export const createFiltersFromValueClickAction = async ({
       })
   );
 
-  return esFilters.mapAndFlattenFilters(filters);
+  return _.uniqWith(esFilters.mapAndFlattenFilters(filters), esFilters.compareFilters);
```

Comment:
by default `esFilters.compareFilters` does a very shallow comparison where it skips some fields inside a filter. 
I don't see any real problems with it in this case, but to be extra safe I'd make it to compare all options:

```
(f1,f2) => esFilters.compareFilters(f1,f2, COMPARE_ALL_OPTIONS)
```